services/src/main/scala/cromwell/services/keyvalue/KeyValueService.scala

```diff
@@ -19,4 +19,3 @@ object KeyValueService {
   case class KvKeyLookupFailed(action: KvGet) extends KvResponse
   case class KvPutSuccess(action: KvPut) extends KvResponse
 }
-
```

Comment:
While I normally appreciate most import optimizations during code edits, I don't like PRs with other changes "cleaning" random files-- causing possible merge conflicts, git blame misattribution, etc. So as a reviewer I'm requesting a revert of this file unless you have actual code to contribute to the diff.